integration-tests/ci-visibility/test/efd-parallel/ci-visibility-test-2.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+const { expect } = require('chai')
```

Comment:
Please only introduce new tests using `node:assert` :)

I am working on changing the current expect calls, so introducing new makes that harder :D